model/commitqueue/commit_queue.go

```diff
@@ -207,3 +207,12 @@ func ClearAllCommitQueues() (int, error) {
 
 	return clearedCount, nil
 }
+
+// EnqueuePRInfo holds information necessary to enqueue a PR to the commit queue.
+type EnqueuePRInfo struct {
```

Comment:
(random comment placement so we can thread)

Note on testing: 
One of the motivations behind this ask was that the PR be closed after it is enqueued using the UI. Can we please test that that happens as expected?

Reply:
Yes - I tested it on [this PR](https://github.com/evergreen-ci/commit-queue-sandbox/pull/419) and observed it was closed when I enqueued using the UI.